topo_processor/cog/execution.py

```diff
@@ -14,16 +10,12 @@ class ExecutionLocal:
 
     @staticmethod
     async def run(cmd: "Command"):
-        start_time = time_in_ms()
```

Comment:
It is a very good idea to have this for when things go wrong, if it was too verbose, maybe switch it down to `trace` level

Reply:
we have reverted this back. let's open another issue for this